images/plbase/plbase-install.sh

```diff
@@ -70,7 +70,7 @@ dnf -y autoremove
 echo "setting up conda..."
 cd /
 arch=`uname -m`
-curl -LO https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-${arch}.sh
+curl -LO https://github.com/conda-forge/miniforge/releases/download/24.3.0-0/Miniforge3-Linux-${arch}.sh
```

Comment:
Can you leave a comment here explaining why this is pinned and which version of Python we're pinning to?

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/10769/commits/e512f97162d40608f880f4542442070a62f47955